buildroot-external/rootfs-overlay/usr/libexec/raucdb-update

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+# shellcheck disable=SC1091
+
+set -e
+
+eval "$(rauc status --output-format=shell)"
+. /etc/os-release
```

Comment:
Nit: We could move that sourcing after the rauc.db check to avoid an unnecessary sourcing.